### PR TITLE
PAE-399.2 - Fix the sass and student_account file paths

### PR DESCRIPTION
**Pull request by luismorenolopera**
Description
---------------
fixes for pearson-vue-theme.

- Change the register.underscore file
- Move the sass files to correct path
- Move student_account folder to correct path

Screenshots:
------------------
**Register page:**

image

### CODE REVIEW

**Review by ivanvgh — APPROVED** [review]
LGTM

File: edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+
+    <button type="submit" class="action action-primary action-update js-register register-button">
+    	<% if ( context.registerFormSubmitButtonText ) { %><%- context.registerFormSubmitButtonText %><% } else { %><%- gettext("Create account") %><% } %>
+    </button>
+</form>
+
+<script type="text/javascript">
+	// This removes all capital letters, except the first one for the label-text class
+	for(label of document.getElementsByClassName('label-text')) {
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore]: for of is not supported by Internet Explorer 11, so it must be changed. @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+		let content = label.textContent.toLocaleLowerCase();
+		content = content.charAt(0).toUpperCase() + content.slice(1);
+		label.textContent = content;
+	}
+
+	var xpath = "//a[text()='Terms of Service and Honor Code']";
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore]: What is this for? @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore (hunk 2 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore]: is the "Terms of Service and Honor Code" link selector, to change for "Terms of Service"

File: edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore (hunk 1 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore]: changed

File: edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    <button type="submit" class="action action-primary action-update js-register register-button">
+    	<% if ( context.registerFormSubmitButtonText ) { %><%- context.registerFormSubmitButtonText %><% } else { %><%- gettext("Create account") %><% } %>
+    </button>
+</form>
+
+<script type="text/javascript">
+	// This removes all capital letters, except the first one for the label-text class
+	var label;
+	for(let i=0; i < document.getElementsByClassName('label-text').length; i++ )  {
```
Comment (Squirrel18) [inline edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore]: Sorry, I forgot, but "let" is partially supported by Internet Explorer, so it must be changed to var. @luismorenolopera

File: edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore (hunk 2 above)
Comment (Squirrel18) [reply edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore]: Is there any other way to do this? Can you point me to the file where this 'Terms of Service and Honor Code' appears? @luismorenolopera

**Review by Squirrel18 — CHANGES_REQUESTED** [review]

File: edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore (hunk 2 above)
Comment (luismorenolopera) [reply edx-platform/pearson-vue-theme/lms/templates/student_account/register.underscore]: https://github.com/Pearson-Advance/edx-platform/blob/pearson-release/juniper.stage/openedx/core/djangoapps/user_authn/views/registration_form.py#L960, changing it here would generate a change in the other sites

**Review by Squirrel18 — APPROVED** [review]

**Pull request merged**